src/pages/DashboardWallet.tsx

```diff
@@ -1,16 +1,19 @@
 import Balance from '../components/dashboard/Balance';
 import TransactionHistory from '../components/dashboard/TransactionHistory';
 import useAuthPublicKey from '../hooks/useAuthPublicKey';
+import useLoadAccount from '../hooks/useLoadAccount';
 
 function DashboardWallet() {
   const { publicKey } = useAuthPublicKey();
+  const { balance, transactions } = useLoadAccount(publicKey as string);
+
   return (
     <main
       className="text-stellar-text-primary bg-stellar-bg-primary"
       data-cy="dashboard-main-container"
     >
-      <Balance publicKey={publicKey as string} />
-      <TransactionHistory />
+      <Balance publicKey={publicKey as string} balance={balance as string} />
```

Comment:
I think you can do:

```typescript
<Balance publicKey balance />
```

Reply:
I can't name it this way as they would be passed as boolean values.